### DISCUSSION TITLE
Installation bug?

### DISCUSSION THREAD

**Discussion by SamSandq** [frang75/nappgui_src#78] [General]
I am trying to install the new version 1.4 to a fresh directory on my Mac.

## WORKS
**git clone --depth 1 https://github.com/frang75/nappgui_src.git**
**cd nappgui_src**

## FAILS
**cmake -G Xcode -S ./src -B ./build**

error message:
   CMake Error: The source directory "/Volumes/SAMSUNG/nappgui_src/src" does not appear to contain CMakeLists.txt.

And indeed it's missing, and on the repo in Github too.

Can I just copy the old one or is it changed?

**Comment by frang75** [comment]
From NAppGUI 1.4, the main CMakeLists.txt is in root dir (not in /src)

**cmake -G Xcode -S . -B build**

Build/install instructions have been updated in https://nappgui.com/en/start/quick.html
